venues/ICLR.cc/2018/Conference/process/submissionProcess.js

```diff
@@ -35,7 +51,8 @@ function() {
       readers: ['everyone'],
       content: {
         authors: ['Anonymous'],
-        authorids: ['Anonymous']
+        authorids: ['Anonymous'],
```

Comment:
can set these authorids to be the authorgroup without having to overwrite it below

Reply:
Actually, there's a potential issue here: the author group needs to get the number of the blinded note, which only happens after it gets posted. We could use the un-blinded note's number, but in UAI, I noticed that it was possible for the blinded and un-blinded note numbers to get off-sync from each other. I don't know if it's worth the risk of that.

Reply:
Yes, the number should be the number of the blinded note, that is how UAI works.

Reply:
Look at the UAI process function: https://github.com/iesl/openreview-scripts/blob/master/venues/auai.org/UAI/2017/process/submissionProcess.js#L60

We create the blinded note and then the author group using the blinded note paper number